### PR TITLE
MAT-6915:  Adding logic to complete the QDM MAT import / re-import

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-6915](https://jira.cms.gov/browse/MAT-6915)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java [hunk 1]
```diff
@@ -224,6 +235,18 @@ public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
     }
   }
 
+  public void checkDuplicateCqlLibraryName(String cqlLibraryName, String measureSetId) {
+    if (StringUtils.isNotEmpty(cqlLibraryName)) {
+      Optional<Measure> anyMatchingMeasure = measureRepository.findByCqlLibraryName(cqlLibraryName);
+      Optional<Measure> matchingMeasureInMeasureSet =
+          anyMatchingMeasure.filter(measure -> measure.getMeasureSetId().equals((measureSetId)));
+      if (anyMatchingMeasure.isPresent() && matchingMeasureInMeasureSet.isEmpty()) {
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/services/MeasureService.java:243]: looks like same as `measureRepository.findByCqlLibraryNameAndMeasureSetIdNot(cqlLibraryName, measureSetId)`

**Review by adongare — APPROVED** [review]

**Review by nmorasb — APPROVED** [review]
